Doc/library/profile.rst

```diff
@@ -148,14 +148,14 @@ entries according to the standard module/line/name string that is printed. The
 :meth:`~pstats.Stats.print_stats` method printed out all the statistics.  You
 might try the following sort calls::
 
-   p.sort_stats('name')
+   p.sort_stats(SortKey.name)
```

Comment:
If people follow the code examples here, SortKey will be undefined.  It needs to be imported.